Geometry/EcalCommonData/plugins/dd4hep/DDEcalBarrelNewAlgo.cc

```diff
@@ -1146,7 +1150,7 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
             }
             unsigned int femCopy(0);
             for (unsigned int ilyFEM(0); ilyFEM != ily.vecIlyFEMZ.size(); ++ilyFEM) {
-              const double radius(ilyRMax - 1_mm - ily.fEMHeightHalf);
+              const double radius(ilyRMax - 1 * dd4hep::mm - ily.fEMHeightHalf);
```

Comment:
@bsunanda , why we need "1 * dd4hep::mm"? May it be "dd4hep::mm"? or compiler is cleaver enough to recognize this?

Reply:
keeping 1 to make it look 1 mm